alien4cloud-core/src/main/java/alien4cloud/deployment/DeploymentTopologyService.java

```diff
@@ -562,11 +580,9 @@ private void checkGroups(Map<String, String> groupsLocationsMapping) {
      * @return
      */
     public Map<String, PropertyValue> processForDeployment(DeploymentTopology deploymentTopology, ApplicationEnvironment environment) {
```

Comment:
This sounds like a huge change and I'm quite afraid of the impacts here.

Previous code was actually changing the deployment topology through injectInputValues meaning that the deployment topology had no more get_input on properties but only values. This method is used just before deployment meaning that the orchestrator plugin didn't had to manage inputs.

The current code does not do that anymore. I am really afraid of consequences here.